src/runtime_src/core/tools/common/tests/TestAIEReconfigOverhead.cpp

```diff
@@ -90,7 +90,12 @@ TestAIEReconfigOverhead::run(std::shared_ptr<xrt_core::device> dev)
   } 
   catch (const std::exception& ex)
   {
-    logger(ptree, "Error", ex.what());
+    std::string err_msg(ex.what());
+    if(err_msg.find(std::string("0xc01e0009")) != std::string::npos) //context creation failed
+      logger (ptree, "Error", "Not enough columns available. Please make sure not other workload is running on the device.");
```

Comment:
"Please make sure **no** other workload is running on the device." Small typo, shows up in the other loggers as well in the other files.